submissions/krylenger/memory-pair-game/app.js

```diff
@@ -0,0 +1,244 @@
+let guessedCards = 0;
+let playedSongs = [];
+let playingCards = [];
+
+const cards = [
+    {
+        key: 1,
+        text: 'orel',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 1,
+        text: null,
+        musicCard: true,
+        song: orelSong
+    },
+    {
+        key: 2,
+        text: 'gopgop',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 2,
+        text: null,
+        musicCard: true,
+        song: gopgopSong
+    },
+    {
+        key: 3,
+        text: 'tanci',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 3,
+        text: null,
+        musicCard: true,
+        song: tanciSong
+    },
+    {
+        key: 4,
+        text: 'cadilac',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 4,
+        text: null,
+        musicCard: true,
+        song: cadilacSong
+    },
+    {
+        key: 5,
+        text: 'dim',
+        musicCard: false,
+        song: null
+    },
+    {
+        key: 5,
+        text: null,
+        musicCard: true,
+        song: dimSong
+    }
+]
+
+const main = document.querySelector('.main');
+let mainGridContainer = document.querySelector('.main__grid-container');
+const modal = document.querySelector('.modal');
+
+
+const styleNewCardFlipperFront = (newCardFlipperFront, card, cardNumber) => {
+    let generatedClassName;
+    switch (cardNumber) {
+        case 0:
+        case 9:
+            generatedClassName = `card--circle-img`;
+            break;
+        case 1:
+        case 2:
+        case 7:
+        case 8:
+            generatedClassName = `card--vertical-line-img`;
+            break;
+        case 3:
+            generatedClassName = `card--left-corner-img`;
+            break;
+        case 6:
+            generatedClassName = `card--right-corner-img`;
+            break;
+        case 4:
+        case 5:
+            generatedClassName = `card--horizontal-line-img`;
+            break;
+        default:
+            generatedClassName = `card--clef-img`;
+    }
+    newCardFlipperFront.classList.add('card__flipper-front', generatedClassName);
+    newCardFlipperFront.setAttribute('data-key', card.key);
+
+}
+
+const createNewCard = (card, cardNumber) => {
+    const newCard = document.createElement('div');
+    const newCardFlipper = document.createElement('div');
+    const newCardFlipperFront = document.createElement('div');
+    const newCardFlipperBack = document.createElement('div');
+    if (card.musicCard) {
+        newCard.classList.add('card', `card--${cardNumber}`, 'musicCard')
+        newCardFlipperBack.classList.add(`card--music-img`);
+    } else {
+        newCard.classList.add('card', `card--${cardNumber}`);
+        newCardFlipperBack.classList.add(`card--${card.text}-img`);
+    }
+    newCardFlipper.classList.add('card__flipper');
+    newCardFlipper.setAttribute('ontouchstart', "this.classList.toggle('card__flipper--flip');")
```

Comment:
You can leave it as it is but it's better to add event listeners by this technique:
`element.addEventListener('ontouchstart', handlerName)`

Reply:
Great! That was the problem it didn't work well on mobile devices!!
I didn't provide the newly created card with callback to set playing logic to the card! 
Awesome!